src/main/java/duke/Duke.java

```diff
@@ -215,6 +215,12 @@ public void run() {
                     String searchTerm = instruction.getParameter();
                     ui.showFoundTasks(tasks.findTasks(searchTerm));
                     break;
+                case ARCHIVE:
+                    Path archiveFilePath = Paths.get("data", "dukeArchive.txt");
```

Comment:
Also you can set up the path of target file relatively at the top of your class to maintain your codes easily later.